### PR TITLE
`ChangingRadioPresetEventArgs::Radio` property

### PR DESCRIPTION
**Pull request by IRacle1**
just pr my funy thing i added in exiled before dildo drama

### CODE REVIEW

**Review by Jesus-QC — APPROVED** [review]

**Review by louis1706 — APPROVED** [review]

File: EXILED/Exiled.Events/EventArgs/Player/ChangingRadioPresetEventArgs.cs [hunk 1]
```diff
@@ -64,5 +71,13 @@ public ChangingRadioPresetEventArgs(Player player, RadioRangeLevel oldValue, Rad
         /// Gets the player who's using the radio.
         /// </summary>
         public Player Player { get; }
+
+        /// <inheritdoc/>
+        public Item Item => Radio;
```
Comment (Vladislav-CS) [inline EXILED/Exiled.Events/EventArgs/Player/ChangingRadioPresetEventArgs.cs:76]: Add an `obsolete` attribute

**Review by Vladislav-CS — CHANGES_REQUESTED** [review]

File: EXILED/Exiled.Events/EventArgs/Player/ChangingRadioPresetEventArgs.cs (hunk 1 above)
Comment (IRacle1) [reply EXILED/Exiled.Events/EventArgs/Player/ChangingRadioPresetEventArgs.cs:76]: Why? It is for implementing `IItemEvent`, and i dont see why it logically should be obsolete

Comment (louis1706) [reply EXILED/Exiled.Events/EventArgs/Player/ChangingRadioPresetEventArgs.cs:76]: i thinks vlad didn't get why this exist lol

Comment (Vladislav-CS) [reply EXILED/Exiled.Events/EventArgs/Player/ChangingRadioPresetEventArgs.cs:76]: Ah, you added it, anyway, there is no need to add it as Radio contains all properties of Item class

Comment (Misfiy) [reply EXILED/Exiled.Events/EventArgs/Player/ChangingRadioPresetEventArgs.cs:76]: Is needed for IItemEvent

Comment (IRacle1) [reply EXILED/Exiled.Events/EventArgs/Player/ChangingRadioPresetEventArgs.cs:76]: - It cant be implemented other way
- i see no problem in that
- 🥱🥱
- its cool
- close review

**Review by Vladislav-CS — APPROVED** [review]

**Pull request merged**